SERVER/templates/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Mmk not a big deal at this point. Thanks for looking! Jinja2 templates aren't bad although sometimes I get irritated working with them.